traffic_ops/testing/api/v4/deliveryservices_test.go

```diff
@@ -47,6 +47,9 @@ func TestDeliveryServices(t *testing.T) {
 			CreateTestDeliveryServicesURLSigKeys(t)
 			GetTestDeliveryServicesURLSigKeys(t)
 			DeleteTestDeliveryServicesURLSigKeys(t)
+			CreateTestDeliveryServicesURISigKeys(t)
```

Comment:
Colloquially these are known as "URI signing keys" as opposed to "URL sig keys", so we should probably fully spell out `Signing` here just to alleviate some confusion.